internal/client/config_test.go

```diff
@@ -22,16 +27,27 @@ const (
 )
 
 func TestValidConfig(t *testing.T) {
-	assert := assert.New(t)
+	require := require.New(t)
+
+	testRootDir, err := os.MkdirTemp("", "flightctl-root")
+	require.NoError(err)
+	defer os.RemoveAll(testRootDir)
```

Comment:
optional: t.TempDir() makes a tmp directory and auto-cleans up

Reply:
Right I had forgotten about that one

Reply:
Good point! Will change.

Reply:
✅ done